src/keybindings.rs

```diff
@@ -129,7 +131,7 @@ pub fn default_emacs_keybindings() -> Keybindings {
     );
     keybindings.add_binding(
         KeyModifiers::CONTROL,
-        Char('y'),
+        Char('g'),
```

Comment:
CTRL-Y is the correct keybinding for paste in emacs. You'll need to pick a different keybinding for redo

Reply:
You can find a list here of some of the standard emacs keybindings: https://caiorss.github.io/Emacs-Elisp-Programming/Keybindings.html

Reply:
@jntrnr - We could reverse and put the redo as ctrl G

Reply:
Sounds good

Reply:
Done